internal/ctrlflow/ctrlflow.go

```diff
@@ -0,0 +1,189 @@
+package ctrlflow
+
+import (
+	"fmt"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"log"
+	mathrand "math/rand"
+	"strconv"
+	"strings"
+
+	"golang.org/x/tools/go/ast/astutil"
+	"golang.org/x/tools/go/ssa"
+	ah "mvdan.cc/garble/internal/asthelper"
+	"mvdan.cc/garble/internal/ssa2ast"
+)
+
+const (
+	FileName = "GARBLE_controlflow.go"
```

Comment:
can we avoid exporting this constant? it feels out of place. the Obfuscate API should return everything that the main package needs.